src/components/layout/Header.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Header() {
+  return (
+    <>
+      <div className="fixed inset-x-0 z-1 h-20 bg-white p-4">
+        <h1 className="text-center text-5xl font-bold">Market</h1>
+      </div>
+    </>
```

Comment:
```suggestion
      <div className="fixed inset-x-0 z-1 h-20 bg-white p-4">
        <h1 className="text-center text-5xl font-bold">Market</h1>
      </div>
```
Request: 불필요한 Fragment는 삭제해주세요.

Reply:
[참고할 GPT 답변](https://chatgpt.com/share/6800afa4-e944-8013-8ae8-eb6a3794e6b9)

Reply:
삭제했습니다!